lab_6_pipeline/pipeline.py

```diff
@@ -78,6 +153,10 @@ def _bootstrap(self) -> AbstractCoNLLUAnalyzer:
         Returns:
             AbstractCoNLLUAnalyzer: Analyzer instance
         """
+        nlp = spacy_udpipe.load_from_path(lang="ru", path=str(UDPIPE_MODEL_PATH))
+        spacy_udpipe.UDPipeModel(lang="ru", path=UDPIPE_MODEL_PATH)
+
+        return AbstractCoNLLUAnalyzer()
```

Comment:
it is a protocol, it does not have implementation

Reply:
Yes, reworked